internal/lock/monitor.go

```diff
@@ -0,0 +1,100 @@
+package lock
+
+import (
+	"sync"
+)
+
+// MonitorFunc is passed to a MonitorVariable. A MonitorFunc receives values on the value chan. If the value is not
```

Comment:
This doesn't look true anymore.

Reply:
True, had forgotten to submit my review but glad you picked it up.